tests/kuttl/tests/ovn_nicmappings/06-cleanup-ovn.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+delete:
```

Comment:
Also need to cleanup net-attach-def created as part of test

Reply:
Done